### PR TITLE
#92 Added Vuex localstorage module

### LINKED ISSUE #92: Vuex module for localstorage
Creating a Vuex module that will work with localstorage, setting and getting token from storage.

### PR DESCRIPTION
**Pull request by markuwas**
Added Vuex localstorage module

### CODE REVIEW

File: src/store/localstorage/getters.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { GetterTree } from 'vuex';
+import { RootStateInterface } from '@/store/types';
+import { TokenStateInterface, TokenInterface } from '@/store/localstorage/types';
+
+export const getters: GetterTree<TokenStateInterface, RootStateInterface> = {
+  isTokenExist(state): boolean {
+    return !!state.token;
```
Comment (what-the-hub) [inline src/store/localstorage/getters.ts]: don't you need to declare getters types in types?

File: src/store/types.ts [hunk 2]
```diff
@@ -2,12 +2,14 @@ import { CommitOptions, DispatchOptions, Store as VuexStore } from 'vuex';
...
 import { UserStateInterface } from '@/store/user/types';
 import { AuthorizationStateInterface } from '@/store/authorization/types';
+import { LocalStorageStateInterface } from '@/store/localstorage/types';
 import { Actions, Getters, Mutations, CooperationStateInterface } from '@/store/cooperation/types';
 
 export interface RootStateInterface {
   cooperationStore: CooperationStateInterface;
   userStore: UserStateInterface;
   authorizationStore: AuthorizationStateInterface;
+  localstorage: LocalStorageStateInterface;
```
Comment (maveric-ui) [inline src/store/types.ts]: localstorage -> localstorageStore

**Review by maveric-ui — DISMISSED** [review]

File: src/store/types.ts (hunk 2 above)
Comment (markuwas) [reply src/store/types.ts]: Fixed

**Review by what-the-hub — DISMISSED** [review]

File: src/store/localstorage/getters.ts (hunk 1 above)
Comment (markuwas) [reply src/store/localstorage/getters.ts]: agree, will be fixed in couple minutes

Comment (markuwas) [reply src/store/localstorage/getters.ts]: fixed

**Review by what-the-hub — APPROVED** [review]

**Review by tsehla42 — APPROVED** [review]

**Pull request merged**